tests/bluetooth/bsim_bt/bsim_test_app/tests_scripts/basic_conn.sh

```diff
@@ -24,12 +24,13 @@ BOARD="${BOARD:-nrf52_bsim}"
 
 cd ${BSIM_OUT_PATH}/bin
 
-Execute ./bs_${BOARD}_samples_bluetooth_peripheral_prj_conf \
-  -v=${VERBOSITY_LEVEL} -s=${SIMULATION_ID} -d=0 -RealEncryption=0
+Execute ./bs_${BOARD}_tests_bluetooth_bsim_bt_bsim_test_app_prj_conf \
+  -v=${VERBOSITY_LEVEL} -s=${SIMULATION_ID} -d=0 -RealEncryption=0 \
+  -testid=peripheral -rs=23
```

Comment:
Just a comment: You do not necessarily need to set the random seed to anything if you don't want. The random seed should be default be set to a value calculated from the device number (-d)